fastjet/internal/predicates/orientation.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package predicates
+
+import (
+	"math/big"
+)
+
+type OrientationKind int
+
+const (
+	// Counterclockwise
+	CCW OrientationKind = iota
+	// Clockwise
+	CW
+	Colinear
+	Indeterminate
+
+	// epsilon determines when the determinant in orientation is too close to 0 to rely on floating point operations
+	// it is 11 * machine epsilon, because there are 11 operations in orientation, with no chained multiplications
+	epsilon = 2.442e-15
```

Comment:
I would replace it with the formula that you describe in the documentation:
```go
epsilon = 11 * macheps
```

also, I would remove `epsilon` from this `const` block and put it on its own (so documentation for `OrientationKind` isn't polluted with `epsilon`.)

actually, I suppose you could move the declaration+definition of `epsilon` directly inside `simpleOrientation` (as it's only used there.)